data/json/npcs/missiondef.json

```diff
@@ -49,7 +49,7 @@
     },
     "start": {
       "effect": [
-        { "npc_add_trait": "ASTHMA" },
```

Comment:
Why this change?

Reply:
Accident! Cut instead of copied. will fix.